graphbolt/src/index_select.h

```diff
@@ -6,11 +6,23 @@
 #ifndef GRAPHBOLT_INDEX_SELECT_H_
 #define GRAPHBOLT_INDEX_SELECT_H_
 
+#include <graphbolt/fused_sampled_subgraph.h>
 #include <torch/script.h>
 
 namespace graphbolt {
 namespace ops {
 
+/** @brief Implemented in the cuda directory. */
+std::tuple<torch::Tensor, torch::Tensor> UVAIndexSelectCSCImpl(
+    torch::Tensor indptr, torch::Tensor indices, torch::Tensor index);
+
+/** @brief Implemented in the cuda directory. */
+std::tuple<torch::Tensor, torch::Tensor> IndexSelectCSCImpl(
+    torch::Tensor indptr, torch::Tensor indices, torch::Tensor index);
+
+std::tuple<torch::Tensor, torch::Tensor> IndexSelectCSC(
+    torch::Tensor indptr, torch::Tensor indices, torch::Tensor index);
```

Comment:
```suggestion
    torch::Tensor indptr, torch::Tensor indices, torch::Tensor column_indices);
```

Reply:
Do you think renaming to `nodes` is a good idea? I just checked the CPU InSubgraph implementation of FusedCSCSamplingGraph and the parameter name is `nodes`.

Reply:
I renamed `index` to `nodes`, let me know if it doesn't fit, then can change to `column_indices`.